config/auspice_config.json

```
@@ -92,6 +92,11 @@
       "type": "ordinal"
     }
   ],
+  "metadata_columns": [
+    "PPX_accession",
+    "INSDC_accession",
+    "restrictedUntil"

```

Hmm, I'm running into an unexpected behavior in the gene builds, from local testing with `nextstrain build . auspice/rsv_a_F_3y.json`.

Looks like the workflow uses the `--color-by-metadata` flag for the gene builds

https://github.com/nextstrain/rsv/blob/770fd86f148af309fc8c145867e79b79c8de6300/workflow/snakemake_rules/export.smk#L45

This overrides the config defined `dataUseTerms` coloring and the field is not included in the metadata. I think it would be safest to always include the `dataUseTerms` field through `metadata_columns`:

```suggestion
    "restrictedUntil",
    "dataUseTerms"
```

---

(This issue already exists before any changes here, e.g. the region coloring is available for the [genome build](https://nextstrain.org/rsv/a/genome/6y?c=region) but not the gene builds, but just focusing on PPX fields here.)

---

Urgh. Good find. Will sort.

---

Nice!

---

Tangent: noticed the entire merged config was getting printed to stderr at the start of validation, so changing that in https://github.com/nextstrain/augur/pull/1878.